canon_host/src/mem_store.rs

```diff
@@ -5,74 +5,21 @@
 // Copyright (c) DUSK NETWORK. All rights reserved.
 
 use std::collections::HashMap;
-use std::fmt;
 use std::sync::Arc;
 
 use parking_lot::RwLock;
 
-use canonical::{
-    ByteSink, Canon, DrySink, Id32, InvalidEncoding, Sink, Source, Store,
-};
-use canonical_derive::Canon;
+use canonical::{Canon, CanonError, Id, IdBuilder, Sink, Source};
```

Comment:
Same for the MemStore, does it make sense still having around?

Reply:
removed